### ISSUE TITLE
Feature Request: Native Login Behaviour

### ISSUE THREAD

**Issue by nicolas-miari** [facebook/facebook-swift-sdk#476]
## Checklist

- [✔︎] I've updated to the latest released version of the SDK
- [✔︎] I've searched for existing feature requests on [GitHub issues](https://github.com/facebook/facebook-swift-sdk/issues)
- [✔︎] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [✔︎] I've given my issue the title: `Feature Request: [name of my feature request]`

## Goals
   Native Login Behavior Support

## Expected Results
  N/A

## Code Samples & Details
My app relies on `FBSDKCoreKit`, `FBSDKShareKit` and `FBSDKLoginKit`. Up until now, I was integrating the Objective-C SDKs using **CocoaPods**. I am moving my dependencies to _Carthage_, so I've replaced the above pods with the corresponding frameworks of this swift.

I have to modify some of the code on my app because some of the API changes. I am having trouble converting this snippet:

    let login = FBSDKLoginManager()
    // Logout to clear cache
    login.logOut()
    login.loginBehavior = .native
    login.logIn(withReadPermissions: ["email"], from: self) { (result, error) in
        // ...

After replacing the pods with the frameworks from this swift SDK, and applying the "Fix-it"s suggested by Xcode, it becomes:

    let login = LoginManager()
    // Logout to clear cache
    login.logOut()
    login.loginBehavior = .native // <- COMPILER ERROR HERE
    login.logIn(permissions: ["email"], from: self) { (result, error) in
        // ...

I get the error:
> Type 'LoginBehavior' has no member 'native'

Looking at the SDK's headers, it is evident that only **browser** login behavior is supported at the moment:

    typedef NS_ENUM(NSUInteger, FBSDKLoginBehavior)
    {
      /**
        This is the default behavior, and indicates logging in via ASWebAuthenticationSession (iOS 12+) or SFAuthenticationSession (iOS 11),
        which present specialized SafariViewControllers. Falls back to plain SFSafariViewController (iOS 9 and 10) or Safari (iOS 8).
       */
       FBSDKLoginBehaviorBrowser = 0,
     } NS_SWIFT_NAME(LoginBehavior);

I take it that there isn't full feature parity yet between the Swift SDK and the Objective-C one...

**Comment by nicolas-miari** [comment]
Well, **this is odd**. I have commented out the offending line:

    login.loginBehavior = .native

The app doesn't seem to behave any different before and after the transition (I've gone back and forth between branches and compared); I get this screen on either case, presented modally from underneath:

![Screen Shot 2019-06-26 at 13 39 48](https://user-images.githubusercontent.com/1000912/60151505-f2ff5400-9817-11e9-8470-d8b72f5be9e9.png)

**Comment by joesus** [comment]
We recently made a change to a tag that was causing an incorrect carthage checkout. Would you mind sharing your Cartfile to help us reproduce?

**Comment by joesus** [comment]
Looked into this some more. Native option is deprecated. Login always goes through browser and will sometimes try to launch the native app from that browser. This is expected behavior. For a variety of reasons we no longer offer the ability to 'force' a native only login. 

Thank you for bringing this to my attention. I've opened a pull request on the ObjC SDK to try and clarify this some more. See: https://github.com/facebook/facebook-objc-sdk/pull/1027 
Closing.
